cmd/lp-cc/cc_test.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/ScottSallinen/lollipop/enforce"
+	"github.com/ScottSallinen/lollipop/framework"
+	"github.com/ScottSallinen/lollipop/graph"
+	"github.com/ScottSallinen/lollipop/mathutils"
+)
+
+func PrintVertexProps(g *graph.Graph[VertexProperty, EdgeProperty, MessageValue], prefix string) {
+	top := prefix
+	sum := uint32(0)
+	for vidx := range g.Vertices {
+		top += fmt.Sprintf("%d:[%d,%d] ", g.Vertices[vidx].Id, g.Vertices[vidx].Property.Value, g.Vertices[vidx].Property.Scratch)
+		sum += g.Vertices[vidx].Property.Value
+	}
+	info(top + " : " + fmt.Sprintf("%d", sum))
+}
+
+// Expectation when 1 is src.
+// TODO: Test other sources!
```

Comment:
Check comments and other spots; you have leftover copy paste in here.

Reply:
Got